Automattic/jetpack
# Search: Set some default configuration values

If you go to the customizer and add a new Jetpack Search widget, it won't show up due to empty default values (no title, unchecked search box, etc.).

Discovered in #8822.

#### Changes proposed in this Pull Request:

Adds a new helper method that's used for both the widget renderer and widget form to supply the default values.

#### Testing instructions:

1. Open the customizer.
2. Add a fresh Jetpack Search widget without touching the form.
3. Verify that the widget shows up in the preview with the search box.
